### PR TITLE
A temporary hack to deal with 'ambiguous use' errors in LLDB REPL

### PR DESCRIPTION
**Pull request by bgogul**
Relevant bug:  [SR-9814](https://bugs.swift.org/browse/SR-9814).

### CODE REVIEW

**Comment by bgogul** [comment]
@swift-ci please test tensorflow macos asan

**Comment by bgogul** [comment]
@swift-ci please test tensorflow asan

**Review by rxwei — APPROVED** [review]

**Review by dan-zheng — APPROVED** [review]

**Comment by rxwei** [comment]
@swift-ci please test tensorflow asan

**Comment by rxwei** [comment]
@swift-ci please test tensorflow Linux

File: lib/Sema/ConstraintSystem.cpp [hunk 1]
```diff
@@ -2045,6 +2045,21 @@ bool ConstraintSystem::salvage(SmallVectorImpl<Solution> &viable, Expr *expr) {
     log << "---Attempting to salvage and emit diagnostics---\n";
   }
 
+  // SWIFT_ENABLE_TENSORFLOW
+  if (DC->getParentModule()->getNameStr().startswith("__lldb_expr") &&
+      viable.size() > 1) {
+    // TOOD(https://bugs.swift.org/browse/SR-9814):
```
[the remark this answers is not in the document]
Comment (bgogul) [reply lib/Sema/ConstraintSystem.cpp]: oops! Thanks, Dan. Sent out https://github.com/apple/swift/pull/22304/

**Pull request merged**